lighting/models/product_source_line_color_temperature_flux.py

```diff
@@ -18,6 +18,8 @@ class LightingProductSourceLineColorTemperatureFlux(models.Model):
 
     nominal_flux = fields.Float(string='Nominal flux',
                         required=True)
+    total_flux = fields.Float(string='Total flux',
```

Comment:
Verify if the old field "total_nominal_flux" is still referenced elsewhere by performing a database search.